app/models/item.rb

```diff
@@ -1,6 +1,11 @@
 class Item < ActiveRecord::Base
   belongs_to :category
 
+  validates :name, presence: true
```

Comment:
I don't know whether it's in vogue now, but there is another way to do this kind of validation

```ruby
validates_presence_of :name, :price, :quantity, :shelf_life_days
```
So if you have the same validation on multiple fields you can do it this way.  The `validates` method is better if you have multiple validations and configurations on one attribute.